package.json

```diff
@@ -18,5 +23,24 @@
   "bugs": {
     "url": "https://github.com/Pomax/react-formbuilder/issues"
   },
-  "homepage": "https://github.com/Pomax/react-formbuilder#readme"
+  "homepage": "https://github.com/Pomax/react-formbuilder#readme",
+  "dependencies": {
+    "babel-core": "^6.22.1",
```

Comment:
Most of these can go into devdeps instead, since you typically only include the already compiled dist library coffee